<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qt</class>
 <widget class="QMainWindow" name="qt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>SimSun-ExtB</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>setting age</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>SimSun-ExtB</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>ageChange[int]</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>SimSun-ExtB</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>age(0~100)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>SimSun-ExtB</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>ageChange[str]</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSlider" name="SliderSetAge">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>SimSun-ExtB</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>setting name</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="lineEdit_3"/>
           </item>
           <item>
            <widget class="QPushButton" name="btnSetName">
             <property name="text">
              <string>set name</string>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/icons/images/check10.png</normaloff>:/icons/images/check10.png</iconset>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="autoRepeatDelay">
              <number>301</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <family>SimSun-ExtB</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>nameChange</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <family>SimSun-ExtB</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>input your name</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_4"/>
         </item>
        </layout>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="btnClose">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>close</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/icons/images/1225473.gif</normaloff>:/icons/images/1225473.gif</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>frame</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
